tests/src/JIT/Performance/CodeQuality/Math/Functions/Abs.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Xunit.Performance;
+
+namespace Functions
+{
+    public static partial class MathTests
+    {
+        private const double absDoubleDelta = 0.0004;
+        private const double absDoubleEpsilon = 2.22e-16;
+        private const double absDoubleExpectedResult = 2499.9999999999659;
+        private const int absIterations = 5000;
+        private const float absSingleDelta = 0.0004f;
+        private const float absSingleEpsilon = 1.19e-07f;
+        private const float absSingleExpectedResult = 2500.03125f;
+
+        [Benchmark]
+        public static void AbsDoubleBenchmark()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    AbsDoubleTest();
+                }
+            }
+        }
+
+        [Benchmark]
+        public static void AbsSingleBenchmark()
+        {
+            foreach (var iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    AbsSingleTest();
+                }
+            }
+        }
+
+        public static void AbsDoubleTest()
+        {
+            var result = 0.0; var value = -1.0;
+
+            for (var iteration = 0; iteration < absIterations; iteration++)
+            {
+                value += absDoubleDelta;
+                result += Math.Abs(value);
+            }
+
+            var diff = Math.Abs(absDoubleExpectedResult - result);
+
+            if ((diff != 0) && (diff < absDoubleEpsilon))
```

Comment:
I think this should just be `if (diff > absDoubleEpsilon)` -- no need to check against zero, and fail if the diff is too large.

Similarly for checks in the other tests.